src/java.desktop/macosx/classes/apple/laf/JRSUIState.java

```diff
@@ -183,8 +183,7 @@ public boolean equals(final Object obj) {
 
         @Override
         public int hashCode() {
-            final long bits = Double.doubleToRawLongBits(value);
-            return super.hashCode() ^ (int)bits ^ (int)(bits >>> 32);
+            return super.hashCode() ^ Double.hashCode(value);
```

Comment:
Since this uses Double.hashCode() not Long.hashCode() it would have been nice to have some comments on that. It took a bit of reading and following to learn that Double.hashCode() first uses Double.doubleToRawLongBits and then calls out to Long.hashCode

Reply:
Double.hashCode is a simple call, and there are no comments in other places where Double.hashCode is used in the JDK source code.